pnpm-lock.yaml

```diff
@@ -19803,6 +19803,14 @@ snapshots:
     transitivePeerDependencies:
       - debug
 
```

Comment:
looks like the peer dependency eace condition is back. can you please try updating the lockfile once more 🙏🏽

Reply:
Updated the lock file, is it better now, not sure what I'm looking for 😄

Reply:
never mind. looks like this PR is fixing the lockfile that was [incorrectly changed in a previous PR](https://github.com/n8n-io/n8n/pull/13664/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL19796). I'm still not sure why this happens 😞